pyvista/core/utilities/geometric_sources.py

```diff
@@ -61,6 +61,212 @@ def translate(surf, center=(0.0, 0.0, 0.0), direction=(1.0, 0.0, 0.0)):
         surf.points += np.array(center, dtype=surf.points.dtype)
 
 
+if _vtk.vtkVersion.GetVTKMajorVersion() >= 9 and 0 <= _vtk.vtkVersion.GetVTKMinorVersion() <= 2:
```

Comment:
A different suggestion to avoid the circular import.  We only support vtk >=9.

```suggestion
if _vtk.vtkVersion.GetVTKMajorVersion() == 9 and _vtk.vtkVersion.GetVTKMinorVersion() <= 2:
```